contrib/vacuumlo/meson.build

```diff
@@ -17,6 +17,10 @@ vacuumlo = executable('vacuumlo',
 )
 contrib_targets += vacuumlo
 
+# if llvm.found()
```

Comment:
Why commented out?

Reply:
I forgot to remove, I noticed this target doesn't have bitcode generated in autoconf setup, it links to frontend libraries and seems not required to have bitcode.
fixup: 578caa8ec54d4a45b712a5ec6b5c78cd5cb222c6